pandas/tests/tools/test_to_datetime.py

```diff
@@ -2501,6 +2501,16 @@ def test_to_datetime_overflow(self):
         with pytest.raises(OutOfBoundsTimedelta, match=msg):
             date_range(start="1/1/1700", freq="B", periods=100000)
 
+    def test_string_invalid_operation(self, cache):
```

Comment:
I didn't know if I should have combined it with  `test_string_na_nat_conversion_malformed`, so I created a different function.